dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/parts/WebpageRepositoryImpl.java

```diff
@@ -491,4 +493,35 @@ protected String getColumnName(String modelProperty) {
         return null;
     }
   }
+
+  @Override
+  public Website getWebsite(UUID rootWebpageUuid) {
```

Comment:
Docstring wäre glaube wichtig hier, die Methode ist nicht selbsterklärend.

Reply:
in WebpageRepository ergänzt (und auch die anderen beiden methoden dokumentiert)